mdoc/Test/en.expected-eii-implementation/CustomNamespace/ClassEnumerator.xml

```diff
@@ -0,0 +1,116 @@
+  <TypeSignature Language="C++ CX" Value="public ref class ClassEnumerator sealed : CustomNamespace::CustomInterface" />
+  <TypeSignature Language="C++ WINRT" Value="class ClassEnumerator sealed : CustomNamespace::CustomInterface" />
+  <AssemblyInfo>
+    <AssemblyName>DocTest-VB-Eii</AssemblyName>
+    <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces>
+    <Interface>
+      <InterfaceName>CustomNamespace.CustomInterface</InterfaceName>
+    </Interface>
+  </Interfaces>
+  <Docs>
+    <summary>To be added.</summary>
+    <remarks>To be added.</remarks>
+  </Docs>
+  <Members>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public ClassEnumerator ();" />
+      <MemberSignature Language="ILAsm" Value=".method public specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
+      <MemberSignature Language="JavaScript" Value="function ClassEnumerator()" Usage="var classEnumerator = new ClassEnumerator();" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; ClassEnumerator();" />
+      <MemberSignature Language="C++ CX" Value="public:&#xA; ClassEnumerator();" />
+      <MemberSignature Language="C++ WINRT" Value=" ClassEnumerator();" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="CustomNamespace.CustomInterface.Prop1">
+      <MemberSignature Language="C#" Value="int CustomNamespace.CustomInterface.Prop1 { get; set; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance int32 CustomProp1" />
+      <MemberSignature Language="VB.NET" Value=" Property Prop1 As Integer Implements CustomInterface.Prop1" />
+      <MemberSignature Language="F#" Usage="CustomNamespace.CustomInterface.Prop1" />
+      <MemberSignature Language="C++ CLI" Value="property int CustomNamespace::CustomInterface::Prop1 { int get(); void set(int value); };" />
+      <MemberSignature Language="C++ CX" Value="property int CustomNamespace::CustomInterface::Prop1 { int get(); void set(int value); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Int32</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
+        <value>To be added.</value>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="CustomNamespace.CustomInterface.Prop2">
```

Comment:
I still think we need to have a new attribute here in order to capture the fact that in the code, it's actually called `CustomProp2` ... there should be a way to detect when these do not match in the case of EII, and add something like `ExplicitInterfaceMemberName="CustomProp2"`, or something.

Think about it from a customer perspective ... if they need information about this member, they will look for `CustomProp2`, so this needs to be a part of the structured data, so that it gets indexed by the search engine accordingly.

Reply:
Done.
Attribute `ExplicitInterfaceMemberName` is added each time when Eii implementation name differs from the original interface name.

A fix is applied in two places in the code to be sure that all cases are covered (initial doc generation and when the update is done).